.changeset/big-pugs-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/core": patch
+---
+
+Replace `lodash-es` by `lodash` library
```

Comment:
Could you please be more descriptive here? Something like:

> Remove `lodash` alias to `lodash-es` to avoid problems with third-party libraries. Replace `lodash-es` with `babel-plugin-lodash` to still get tree shaking.